libshvchainpack/include/shv/chainpack/accessgrant.h

```diff
@@ -55,5 +55,23 @@ struct SHVCHAINPACK_DECL_EXPORT UserLogin
 	chainpack::RpcValue toRpcValueMap() const;
 	static UserLogin fromRpcValue(const RpcValue &val);
 };
+
+struct SHVCHAINPACK_DECL_EXPORT AccessGrant
+{
+	using AccessLevel = MetaMethod::AccessLevel;
+
+	int accessLevelInt = 0;
```

Comment:
Is it mandatory for this to be an int? Can't it be `AccessLevel`?

Reply:
No, RpcMessage can have any int from <0,63> assigned to accessLevel tag.